README.ja.md

```diff
@@ -242,7 +242,7 @@ Limaにはデータの喪失を引き起こすバグが含まれているかも
 はい。動くでしょう。ただ、ARM Macでの定期的なテストはなされていません(CIがないため)。
 
 #### "Ubuntu以外のゲストを動かすことはできますか？"
-Alpine, Arch Linux, Debian, Fedora, openSUSE, Rocky が動くことが知られています。[`./examples/`](./examples/)を見てください。
+AlmaLinux, Alpine, Arch Linux, Debian, Fedora, openSUSE, Rocky が動くことが知られています。[`./examples/`](./examples/)を見てください。
```

Comment:
Japanese translation should be a separate commit.
Please update the revision/date in the header too.